cmd/dagger/engine.go

```diff
@@ -97,9 +97,6 @@ func withEngine(
 
 		params.WithTerminal = withTerminal
 
-		params.Stdin = os.Stdin
-		params.Stdout = os.Stdout
```

Comment:
were these 2 lines added when MCP was added? 

i'm not sure the downstream effect of this deletion given that other changes are tightly scoped to `dagger mcp` and the set env vars

Reply:
yes

Reply:
I could indeed put the withEngine change separately, but this was part of the refactor as only MCP uses it for now and it's a way to make sure only MCP has the session attachable.